### DISCUSSION TITLE
Avatars and logos deform

### DISCUSSION THREAD

**Discussion by kozybear** [HEIGE-PCloud/DoIt#1173] [Q&A] [answered]
Hi everybody ! I am currently using the DoIt theme and encounter this problem. Before, both the logo and Avatar were fine, but today when I looked at it again, they became like below. Does anyone have a way to fix this problem? Thanks everyone .
This is my repo : https://github.com/[h4sh1334r/h4sh1334r.github.io](https://github.com/h4sh1334r/h4sh1334r.github.io)
![Screenshot 2024-01-31 105121](https://github.com/HEIGE-PCloud/DoIt/assets/130491266/5ff39400-6da5-4c2b-a6d8-75ba928028a3)

**Comment by HEIGE-PCloud** [comment, marked as answer]
See #1165 

It’s fixed in the latest commits, so update the theme should resolve it :)

**Reply by kozybear** [reply]
Thank you so much. I fixed it .
